tests/minimax.wcon

```diff
@@ -0,0 +1,39 @@
+{
+    "tracker-commons":true,
+    "metadata":{
+           "lab":{"location":"CRB, room 5020", "name":"Behavioural Genomics" },
+           "who":"Firstname Lastname",
+           "timestamp":"2012-04-23T18:25:43.511Z",
+           "temperature":{ "experiment":22, "cultivation":20, "units":"C" },
+           "humidity":{ "value":40, "units":"%" },
+           "dish":{ "type":"petri", "size":35, "units":"mm" },
+           "food":"none",
+           "media":"agarose",
+           "sex":"hermaphrodite",
+           "stage":"adult",
+           "age":"18:25:43.511",
+           "strain":"CB4856",
+           "image_orientation":"imaged onto agar",
+           "protocol":"text description of protocol",
+           "software":{
+                "tracker":{ "name":"Software Name", "version":"1.3.0"},
+                "featureID":"@OMG"
+           },
+           "settings":"Any valid JSON entry with hardware and software configuration can go here"
+     },
+    "units":{"t":"s", "x":"mm", "y":"m"},
+    "data":[
+        {"@OMG":{
+               "feature_names":["speed", "curvature", "width"],
+               "feature_units":["mm/s", "1/mm", "mm"]
+        }},	
+        { "id":4, "t":1.4, "x":[1215.11, 1216.14, 1217.12], "y":[234.89, 265.23, 235.08] },
+	    { "id":1, "t":[1.3,1.5], "x":[[1,1,1],[1215.11, 1216.14, 1217.12]], 
+	                                 "y":[[2,2,2],[234.89, 265.23, 235.08]], "ox":[5000,5001] },
+		{ "id":2, "t":1.4, "x":[125.11, 126.14, 117.12], "y":[23.3, 22.23, 21135.08] },
+		{ "id":1, "t":1.4, "x":[1215.11, 1216.14, 1217.12], "y":[234.89, 265.23, 235.08]},
+		{ "id":1, "t":1.3, "x":[1, 1, 1], "y":[2, 2, 2], "@OMG":{"feature_values":[55,55,55]}},
+		{ "id":2, "t":1.5, "x":[1215.11, 1216.14, 1217.12], "y":[234.89, 265.23, 235.08] }
+	],
+	"@AQ": 5
```

Comment:
I think your comment here might be moot if you accept my previous comment's recommendations to use only top-level custom features.